.github/workflows/tests.yml

```
@@ -1,5 +1,9 @@
 name: Tests
-on: [push, pull_request]
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: '30 6 * * *'
```

@ZedThree It is only one CI job thus I thought daily would gives us very few PRs to look at.
But we can only run it weekly, at which case there are probably like ~10 PRs to go through.
I would rather use computer time but maintainer time.
```suggestion
    # run sunday moring
    - cron: '30 6 * * 0'
```